chromium_src/content/public/common/url_utils.cc

```diff
@@ -0,0 +1,76 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "url/gurl.h"
+
+namespace {
+
+const char kBraveUIBadCastCrashURL[] = "brave://badcastcrash/";
+const char kBraveUICheckCrashURL[] = "brave://checkcrash/";
+const char kBraveUICrashURL[] = "brave://crash/";
+const char kBraveUIDumpURL[] = "brave://crashdump/";
+const char kBraveUIHangURL[] = "brave://hang/";
+const char kBraveUIKillURL[] = "brave://kill/";
+const char kBraveUIMemoryExhaustURL[] = "brave://memory-exhaust/";
+#if defined(OS_WIN)
+const char kBraveUIHeapCorruptionCrashURL[] = "brave://heapcorruptioncrash/";
+#endif
+#if defined(ADDRESS_SANITIZER)
+const char kBraveUICrashHeapOverflowURL[] = "brave://crash/heap-overflow";
+const char kBraveUICrashHeapUnderflowURL[] = "brave://crash/heap-underflow";
+const char kBraveUICrashUseAfterFreeURL[] = "brave://crash/use-after-free";
+
+#if defined(OS_WIN)
+const char kBraveUICrashCorruptHeapBlockURL[] =
+    "brave://crash/corrupt-heap-block";
+const char kBraveUICrashCorruptHeapURL[] = "brave://crash/corrupt-heap";
+#endif  // OS_WIN
+#endif  // ADDRESS_SANITIZER
+
+#if DCHECK_IS_ON()
+const char kBraveUICrashDcheckURL[] = "brave://crash/dcheck";
+#endif
+
+const char kBraveUIShorthangURL[] = "brave://shorthang/";
+
+bool IsBraveRendererDebugURL(const GURL& url) {
+  if (url == kBraveUICheckCrashURL || url == kBraveUIBadCastCrashURL ||
```

Comment:
I don't think this is the change we want anyway, but why would we do all this when we could just convert chrome -> brave and call `IsRendererDebugURL`?